script/cms/genGraphQl.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generate the node files from the grapQl output in the pages.json file
+ * so we can compare them to the node files that we generate
+ */
+
+const _ = require('lodash');
+const fs = require('fs');
+const Conf = require('./config');
+
+const conf = new Conf();
+
+const graphqlDir = conf.nodeFileDir; // where we put the graphql nodes
+
+/*
+* Save a single node to file
+*/
+
+function saveNode(node) {
+  // Create the dir if it doesn't exist
+  if (!fs.existsSync(graphqlDir)) {
+    fs.mkdirSync(graphqlDir);
+  }
```

Comment:
Alternatively, we can use [`ensureDirSync`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir-sync.md) if we use `fs-extra`.
```suggestion
  fs.ensureDirSync(graphqlDir);
```
However, if we use [`fs.outputJsonSync()`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson-sync.md), we don't need this bit at all, as it'll ensure the directories are created.